File: fixtures/runtime-parity/indexed-write-path1-parity/solution.cs
using System.Collections.Generic;

public class Solution
{
    public int solve(int[] nums)
    {
        var freq = new Dictionary<int, int> { [1] = 2 };
        int left = nums[0];
        freq[left] = freq[left] - 1;
        return freq[left];
    }
}

File: fixtures/runtime-parity/indexed-write-path1-parity/solution.java
import java.util.*;

class Solution {
  public int solve(int[] nums) {
    Map<Integer, Integer> freq = new HashMap<>();
    freq.put(1, 2);
    int left = nums[0];
    freq.put(left, freq.get(left) - 1);
    return freq.get(left);
  }
}
